network/load-balancer/quickstart.mdx

```diff
@@ -6,78 +6,86 @@ content:
   h1: Load Balancers - Quickstart
   paragraph: This page shows you how to get started with Scaleway Load Balancers.
 steps:
-  - step: TODO
-    type: HowToStep
-    url: https://www.scaleway.com/en/docs/network/load-balancer/quickstart
 dates:
   validation: 2022-11-03
   posted: 2021-05-26
 ---
 
 [Load Balancers](https://www.scaleway.com/en/load-balancer/) are highly available and fully-managed Instances which allow you to distribute your workload across your various servers. They ensure the scaling of all your applications while securing their continuous availability, even in the event of heavy traffic. Load Balancers are commonly used to improve the performance and reliability of web sites, applications, databases and other services.
 
+<Lightbox src="scaleway-load-balancer.webp" alt="" />
+
 <Message type="requirement">
   - You have an account and are logged into the [Scaleway console](https://console.scaleway.com) 
 </Message>
 
 ## How to create a Load Balancer
 
 1. Click **Load Balancers** in the **Network** section of the [Scaleway Console](https://console.scaleway.com) side menu. If you have not already created a Load Balancer, the product creation page is displayed.
+
 2. Click **Create a Load Balancer**. The creation wizard displays.
-3. Complete the following steps in the wizard: 
-    - Enter the Load Balancer **name** and **description**. Optionally, you can assign **tags** to help organize your Load Balancers.
-    - Choose the **Availability Zone** in which your Load Balancer will be deployed geographically. Currently we provide the following Availability Zones:
+
+3. Enter a **name** and **description** for your Load Balancer. Optionally, you can assign **tags** to help organize your Load Balancers.
+    <Lightbox src="scaleway-load-balancer-name.webp" alt="" />
+
+4. Choose the **Availability Zone** in which your Load Balancer will be deployed geographically. Currently we provide the following Availability Zones:
```

Comment:
Can you please update the AZ list as we have already added AMS2 since this document was last updated. I also think we should rename Warsaw to Warsaw 1 since this is how it is displayed in the console now.
Please note we will have to update this page really soon with WAW2 availability.

Reply:
done